.circleci/config.yml

```diff
@@ -25,8 +25,7 @@ jobs:
 
 
   release-charts:
-    machine:
-      image: ubuntu-2004:202201-02
+    runs-on: ubuntu-latest
```

Comment:
Can we use the same version for both? Either latest or specific is fine with me

Reply:
Oops - accidentally left one when trying something. Both use the same image now 👍🏻